src/App.tsx

```diff
@@ -1,61 +1,143 @@
 import './App.scss';
+import React, { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { getUserById } from './services/user';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(
+    ...todos.map(todo => todo.id),
+  );
+
+  return maxId + 1;
+}
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+  const [title, setTitle] = useState<string>('');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
```
it's redundant to specify primitive types

Reply:
done.